awpy/types.py

```diff
@@ -569,6 +569,9 @@ class GameRound(TypedDict):
     winningTeam: str | None
     losingTeam: str | None
     roundEndReason: str
+    mvpName: str | None
```

Comment:
If we do NOT make them pointers on the go side then we can remove the `| None` here.

I still think these should be null of no player exists for the MVP announcement but if we do not want to go that route than we should still be consistent and remove it here.